spec/lib/bank_accounts/account_spec.rb

```diff
@@ -0,0 +1,74 @@
+require 'spec_helper'
+
+describe BankAccounts::Account do
+
```

Comment:
Note: If you wanted to avoid having to say `BankAccounts::Account` every time in your tests, you can wrap the entire file in `module BankAccounts` (the same way you wrapped the class you're testing) and everything should be properly namespaced.